### ISSUE TITLE
nodetool repair won't start 

### ISSUE THREAD

**Issue by l15k4** [scylladb/scylla#2631]
Cluster size: 3 x i3.xlarge
AMI for ec2 i3 instances, 1.7.0rc3
Replication factor 3

I was running repair on this cluster many times without a problem until today. Starting repair of ` 141.35 GB` simply doesn't do anything, I can't see any activity at all on any node in Grafana.
```
$ nodetool status
Datacenter: datacenter1
=======================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address        Load       Tokens       Owns    Host ID                               Rack
UN      141.35 GB  256          ?       ccda8c3f-ec55-4020-9591-945fad0c100c  rack1
UN      184.49 GB  256          ?       e1465f3a-9d8b-48ef-9b16-4c6ff5557c50  rack1
UN      165.21 GB  256          ?       bfd7ba99-6e20-454c-8263-d0c7ef349b2c  rack1

Note: Non-system keyspaces don't have the same replication settings, effective ownership information is meaningless
$ nodetool repair gwiq
[2017-07-25 13:49:26,962] Starting repair command #2, repairing 1 ranges for keyspace gwiq (parallelism=SEQUENTIAL, full=true)

$ nodetool drain
$ sudo service scylla-server restart
Redirecting to /bin/systemctl restart  scylla-server.service
$ nodetool repair gwiq
[2017-07-25 13:54:43,681] Starting repair command #1, repairing 1 ranges for keyspace gwiq (parallelism=SEQUENTIAL, full=true)
```
```
Jul 25 12:51:39 ip-172-31-43-170.eu-west-1.compute.internal scylla[4270]:  [shard 0] repair - starting user-requested repair for keyspace system_traces, repair id 1, options {trace=false, primaryRange=false, jobThreads=1, parallelism=paral
Jul 25 13:49:26 ip-172-31-43-170.eu-west-1.compute.internal scylla[4270]:  [shard 0] repair - starting user-requested repair for keyspace gwiq, repair id 2, options {trace=false, primaryRange=false, jobThreads=1, parallelism=parallel, incr
```

Doing this ^^ simply doesn't initiate repairing, I tried to restart Scylla server but it didn't help. The repair process never ends, it is running indefinitely without doing anything.

**Comment by tzach** [comment]
@asias @amnonh any idea?

**Comment by l15k4** [comment]
Hey,

the way it ended : 
1) I drained the `165.21 GB`  node and restarted it. It caused it to have the exact same size `141.35 GB` as the other node.
2) now I have unfortunately 2x `141.35 GB` and one `184.49 GB` ... this happened because those 2 scylla nodes crashed a few times and the ec2 instance became inaccessible so I had to reboot it and I think the whole commit log got lost.

Please what is the best way to recover those 2x `141.35 GB` nodes from the `184.49 GB` one? Should I `rebuild` them from it? Like, deleting data and doing rebuild on both nodes? Repair doesn't work in this case unfortunately.

**Comment by tzach** [comment]
@l15k4 can you jump on the user slack for this?

**Comment by l15k4** [comment]
Hey @tzach, it is resolved. Note that these sizes were taken after full compaction and roughly corresponded to `du -sh *` sizes.

In the end, I resolved it by stopping one `141.35GB` node, and repaired the other one from the full one, and then started the node again and repaired it from those 2 full nodes, so it seems to me like if the repair used a Quorum policy, at least in this case.

I upgraded to 1.7.3 recently so I would rather let you know about further progress next time if you don't mind.
